pkg/api/types.go

```diff
@@ -597,10 +597,34 @@ const (
 	ClaimLost PersistentVolumeClaimPhase = "Lost"
 )
 
+type HostPathType string
+
+const (
+	// For backwards compatible, leave it empty if unset
+	HostPathUnset HostPathType = ""
```

Comment:
What does this actually mean?  If we find a volume that has this value, what behavior is triggered?  I don't think we need this at all.  If we document the default in v1 and we have the defaulting logic in place, we should never see this happen.